### ISSUE TITLE
obsolete num_quad_lloyd_steps keyword in call to pygmsh.generate_mesh

### ISSUE THREAD

**Issue by gdmcbain** [geodynamics/specfem3d#1292]
I'm not using specfem3d myself but am following up bug report nschloe/pygmsh#223, about specfem3d EXAMPLES/Gmsh_simple_lddrk failing.

I have installed specfem3d and when I run `mesh_example_with_gmsh.py`, it raises (from [L386](https://github.com/geodynamics/specfem3d/blob/master/EXAMPLES/Gmsh_simple_lddrk/mesh_example_with_gmsh.py#L386))
```
TypeError: generate_mesh() got an unexpected keyword argument 'num_quad_lloyd_steps'
```

Indeed, pygmsh shed this keyword in March: https://github.com/nschloe/pygmsh/commit/8a7592539fbf2c33d0b2b5e711e522293caad1e0#diff-f5350ec7a250a8f9f1ea9a189d3f51fa

I think it should be safe enough just to delete the keyword here.

I doubt this is the issue with nschloe/pygmsh#223 though; I'll press on with that investigation.

**Comment by nschloe** [comment]
Btw all the smoothing functionality is now improved in [optimesh](https://github.com/nschloe/optimesh). -- A definite recommendation for gmsh meshes which by default aren't of the highest quality.

**Comment by gdmcbain** [comment]
Fixed by #1295 .
